### DISCUSSION TITLE
RSS Downloader not loading until RSS feeds are loaded

### DISCUSSION THREAD

**Discussion by ChriZathens** [Novik/ruTorrent#2824] [Q&A]
Hello guys!
I wanted to ask if something that is happening is expected.
Since a few weeks (I believe after the commits that dealt with refactoring the RSS feeds a bit, but perhaps it is just a coincidence), when I open the web interface, If I click the RSS downloader, I don't see the feeds I have set up to download autpomatically. I only see an option to add a feed.
It is only after the feeds are loaded that the downloader also opens "normally" and shows what I have already configured.
Before the feeds are loaded, the only way to open the RSS Downloader is by right-clicking in "All Feeds" and select RSS Manager.
Is this expected? Is this also happening to anyone else, or is it just a me problem?
Just for FYI I am using the latest built from github (5.1.3)
Thanks !!!!

**Comment by ranirahn** [comment]
That is intended. When feeds are not loaded you cant select them also in RSS Manager. So changing anything in there when Feeds are not loaded could be bad. There was a time when RSS Manager did not open at all exept right click on Feeds and open it from there, but that was fixed. For my opinion the RSS Downloader button should always open the RSS Manager, but I understand the problem when Feeds are not loaded yet.
